homeassistant/components/wiz/config_flow.py

```diff
@@ -0,0 +1,55 @@
+"""Config flow for WiZ Platform."""
+import logging
+
+from pywizlight import wizlight
+from pywizlight.exceptions import WizLightConnectionError, WizLightTimeOutError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.data_entry_flow import AbortFlow
+
+from .const import DEFAULT_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
+    }
+)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for WiZ."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        errors = {}
+        if user_input is not None:
+            bulb = wizlight(user_input[CONF_HOST])
+            try:
+                mac = await bulb.getMac()
+            except WizLightTimeOutError:
+                errors["base"] = "bulb_time_out"
+            except ConnectionRefusedError:
+                errors["base"] = "cannot_connect"
+            except WizLightConnectionError:
+                errors["base"] = "no_wiz_light"
+            except AbortFlow:
+                return self.async_abort(reason="single_instance_allowed")
```

Comment:
This is an except guarding a call into the pywizlight library. It will not be able to throw AbortFlow.

```suggestion
```

And since there is a config entry per light, I also don't think that you want to limit this integration to a single instance.